src/c++/perf_analyzer/concurrency_worker.h

```diff
@@ -139,7 +140,12 @@ class ConcurrencyWorker : public LoadWorker {
 
   uint32_t GetCtxId();
 
-  virtual size_t GetNumActiveThreads() override { return active_threads_; }
+  void CreateContextPost(std::shared_ptr<InferContext> ctx) override
```

Comment:
What does "post" mean in this context?

Reply:
this is stale. No post functions. There is a `finalize` now, which I believe is documented what it does.